README.md

```diff
@@ -3,7 +3,7 @@
 [![Build Status](https://travis-ci.org/tgstation/-tg-station.png)](https://travis-ci.org/tgstation/-tg-station)
 
 
-**Website:** http://www.tgstation13.org <BR>
+**Website:** http://wiki.baystation12.net/Tajaran_Culture <BR>

```

Comment:
That's the one change he made so he could have a PR up to have #10000. He didn't actually have an idea, and he forgot to revert this placeholder.